### PR TITLE
Move metaverse server URL info into central files for easier updating

### PR DESCRIPTION
**Pull request by Misterblue**
Move metaverse server URLS to a single files (one for C++ and one for JS). This makes it easier to update the URL to the metaverse server rather than relying on scattered constants.

This does not replace all references to HiFi but updates only the code that needs to access the metaverse server. Other URL references are untouched.

This is a first step to making the server address configuration dynamic. First constants in one place and eventually this will be updated to set from parameters/configFiles at runtime.

### CODE REVIEW

**Comment by Misterblue** [comment]
I didn't mention it but... this PR should make no operational changes. This just moves URL definitions to different file locations.

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-d6746e3_f26c7d3c.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-d6746e3.tar_f26c7d3c.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-d6746e3.tar_f26c7d3c.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-d6746e3_f26c7d3c.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-d6746e3_f26c7d3c.exe

File: libraries/ktx/src/khronos/KHR.h [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #define khronos_khr_hpp
 
 #include <unordered_map>
+#include <stdexcept>
```
Comment (daleglass) [inline libraries/ktx/src/khronos/KHR.h:14]: This seems unnecessary. I think this is probably a fix for a separate issue (likely a compiler being more strict than before) and should go in its own PR.

File: libraries/shared/src/shared/Storage.h [hunk 2]
```diff
@@ -14,6 +14,7 @@
 #include <vector>
 #include <memory>
 #include <functional>
+#include <stdexcept>
```
Comment (daleglass) [inline libraries/shared/src/shared/Storage.h:17]: Same here.

File: script-archive/lobby.js [hunk 3]
```diff
@@ -153,7 +153,7 @@ var places = {};
 
 function changeLobbyTextures() {
   var req = new XMLHttpRequest();
-  req.open("GET", "https://metaverse.highfidelity.com/api/v1/places?limit=21", false);
+    req.open("GET", URLs.METAVERSE_URL + "/api/v1/places?limit=21", false);
```
Comment (daleglass) [inline script-archive/lobby.js:156]: Extra space here

File: libraries/shared/src/shared/Storage.h (hunk 2 above)
Comment (Misterblue) [reply libraries/shared/src/shared/Storage.h:17]: I found that <stdexcept> was needed for compiling the project on Windows10 with Visual Studio 2019 and using the cmake that is included in VS. It obviously is not needed under Linux.

File: libraries/ktx/src/khronos/KHR.h (hunk 1 above)
Comment (Misterblue) [reply libraries/ktx/src/khronos/KHR.h:14]: I found that the include of stdexcept was needed for compiling the project on Windows10 with Visual Studio 2019 and using the cmake that is included in VS. It obviously is not needed under Linux.

**Comment by vegaslon** [comment]
all the javascript files changed need a require to shared.js

**Comment by Misterblue** [comment]
None of the changed JavaScript files have any 'required' statements in them. Is there a JavaScript environment builder that needs to be updated with the necessary references to shared.js?
As it stands, Interface builds and runs (using these URL references) but there could be paths that have not been tested for undefined at runtime variables.

**Comment by vegaslon** [comment]
yes none of the javascript files are pulling from shared.js for variables and methods so they do not work at all with this change. shared.js was was only for the domain servers web interface.

**Comment by thoys** [comment]
build on Windows seems to fail with error:
```
D:\a\project-athena\project-athena\libraries\shared\src\shared\Storage.cpp(65,37): error C3861: 'runtime_error': identifier not found [D:\a\project-athena\build\libraries\shared\shared.vcxproj]
```

**Comment by Misterblue** [comment]
The error building on Windows in Storage.cpp would be caused by the missing "include <stdexcept>" in Storage.h. The 'include' was moved to a separate PR.
Wonder why it compiled before.

**Comment by vegaslon** [comment]
it is the commit they reverted last night

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-dfe9bab_4b27f80c.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-dfe9bab.tar_4b27f80c.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-dfe9bab.tar_4b27f80c.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-dfe9bab_4b27f80c.sh

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-9f05fc0_0bc62965.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-9f05fc0_0bc62965.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-9f05fc0.tar_0bc62965.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/ProjectAthena-Alpha-PR411-9f05fc0.tar_0bc62965.Z

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-5646377_0bc62965.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-5646377.tar_0bc62965.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-5646377.tar_0bc62965.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-5646377_0bc62965.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-5646377_0bc62965.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-0cb7645_e8c245ca.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-0cb7645_e8c245ca.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-0cb7645.tar_e8c245ca.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-0cb7645.tar_e8c245ca.gz

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-fbb4ae9_ad1825c2.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-fbb4ae9_ad1825c2.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-fbb4ae9.tar_ad1825c2.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-fbb4ae9.tar_ad1825c2.Z

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-c735346_ef056ecf.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-c735346_ef056ecf.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-c735346.tar_ef056ecf.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-c735346.tar_ef056ecf.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR411/Vircadia-Alpha-PR411-c735346_ef056ecf.exe
